azure-pipelines/ui-automation/templates/flank/flank.yml

```diff
@@ -1,19 +1,27 @@
 flank:
-  ## Max Test Shards
-  # test shards - the amount of groups to split the test suite into
-  # set to -1 to use one shard per test. default: 1
+   ## Max Test Shards
+   # test shards - the amount of groups to split the test suite into
+   # set to -1 to use one shard per test. default: 1
    max-test-shards: 5
 
-  ## Number of Test Runs
-  # test runs - the amount of times to run the tests.
-  # 1 runs the tests once. 10 runs all the tests 10x
+   ## Number of Test Runs
+   # test runs - the amount of times to run the tests.
+   # 1 runs the tests once. 10 runs all the tests 10x
    num-test-runs: 1
 
-  ## Use Average Test Time for New Tests flag
-  # Enable using average time from previous tests duration when using SmartShard and tests did not run before.
-  # Default: false
+   ## Use Average Test Time for New Tests flag
+   # Enable using average time from previous tests duration when using SmartShard and tests did not run before.
+   # Default: false
    use-average-test-time-for-new-tests: true
 
-  ## Local Results Directory
-  # Local folder to store the test result. Folder is DELETED before each run to ensure only artifacts from the new run are saved.
-   local-result-dir: flank
\ No newline at end of file
+   ## Local Results Directory
+   # Local folder to store the test result. Folder is DELETED before each run to ensure only artifacts from the new run are saved.
+   local-result-dir: flank
+
+   ## Output Style flag
+   # Output style of execution status. May be one of [verbose, multi, single, compact].
+   # For runs with only one test execution the default value is 'verbose', in other cases
+   # 'multi' is used as the default. The output style 'multi' is not displayed correctly on consoles
+   # which don't support ansi codes, to avoid corrupted output use single or verbose.
+   # The output style `compact` is used to produce less detailed output, it prints just Args, test and matrix count, weblinks, cost, and result reports.
+   output-style: single
```

Comment:
Was this also the default?

Reply:
The default was `verbose` in this case, which was producing a lot of output that I felt was unnecessary.